openassessment/assessment/test/test_staff.py

```diff
@@ -57,6 +57,18 @@ def _peer_assess(sub, scorer_id, scores):
         ('ai', lambda sub, scorer_id, scores: TestStaffAssessment._ai_assess(sub))
     ]
 
+    def _verify_done_state(self, uuid, requirements, done=True):
```

Comment:
Sorry to keep filing nits on this helper method, but I would not make "done" an optional parameter. It isn't clear what the method is testing if you don't pass in the expected value. I would call the parameter "expected_done" or "expected_state".